src/tests/test_cases/test.py

```diff
@@ -290,6 +290,20 @@ def first_word_has_unicode_byte():
         """あy."""
 
 
+if sys.version_info[0:2] != (3, 5):
```

Comment:
This is still failing. I think you need to add an extra file for f-string and only load it if we're with Python 3.6+,